columnphysics/icepack_isotope.F90

```diff
@@ -482,10 +482,6 @@ function wiso_alpi(isp,tk)
 
       wiso_alpi = exp(alpai(isp)/tk**2 + alpbi(isp)/tk + alpci(isp))
 
-#ifdef NOFRAC
-      wiso_alpi = c1
-#endif
```

Comment:
This is the only instance of "NOFRAC"... was this added for compiling CICE as part of another model ? (@eclare108213 you added that in 1ae0446 (Isotopes for Icepack (#307), 2020-04-06))

Reply:
According to @dabail10, this was added for debugging the isotopes and is not needed at this time.